app/pages/project/index.jsx

```diff
@@ -86,23 +88,17 @@ class ProjectPageController extends React.Component {
       }
     }
   }
+
   componentWillUnmount() {
     Split.clear();
   }
 
-  setupSplits(props) {
-    if (!props) {
-      ({ props } = this);
-    }
-    const { user } = props;
-    const { owner, name } = props.params;
-
+  setupSplits(slug, user) {
     if (user) {
-      Split.load(`${owner}/${name}`)
+      return Split.load(slug)
       .then((splits) => {
```

Comment:
The name `splits` implies an array, but it's actually a single object. Is it worth changing that to `split` throughout the code to make it clearer that it's a single thing?

Reply:
Feel free to ignore this if `splits` is actually a hash of splits keyed by split type/name.

Reply:
It is indeed a hash of keyed splits. In theory, a project can have many experiments running so many splits could be returned. In practice, projects usually a single experiment at a time.